src/sgame/sg_weapon.cpp

```diff
@@ -40,7 +40,7 @@ static bool TakesDamages( gentity_t const* ent )
 {
 	return false
 		|| Entities::IsAlive( ent )
-		|| ( ent && ent->s.eType == entityType_t::ET_MOVER )
+		|| ( ent && ent->s.eType == entityType_t::ET_MOVER && ent->health > 0 )
```

Comment:
Would `https://github.com/Unvanquished/Unvanquished/blob/master/src/sgame/Entities.cpp#L82` be a better check?

Reply:
No, because movers are *not* in the CBSE, and so do not have components